.github/ISSUE_TEMPLATE/feature-request.md

```diff
@@ -1,17 +1,30 @@
 ---
 name: "\U0001F4A1 Feature request \U0001F4A1"
 about: Suggest an idea for this project
+title: ''
+labels: ['feature', 'needs-triage']
```

Comment:
Tested this in my own branch. The label wasn’t auto absagend. I think this is because the label is called `enhancement` not `feature`.

Reply:
That’s right, the labels have to be created first. Or change the template to use existing labels. But the labels are correctly assigned once they are available.

Reply:
Changed it to enchancement (this is the label that is already there indeed)